CSharp/BatchExplorer/Views/MainView.xaml

```diff
@@ -687,12 +658,20 @@
                                         <TextBlock HorizontalAlignment="Center" Text="Delete" FontSize="12" Foreground="Black" />
                                     </StackPanel>
                                 </Button.Content>
-                            </Button>                            
-                        </StackPanel>
+                            </Button>
+                            <TextBox x:Name="SearchJobsTxtBox" 

```

Comment:
I agree with (2) in that feature would be nice and I have that implemented in other unrelated code elsewhere (it's nice to have). That needs more coding / testing though because ideally the filter would operate client side (fast search) w/o needing to refresh the jobs server-side. It would also be nice to add a filter (server-side) for only say Active Jobs to operate in concert with the client side filter